diff --git a/node_modules/chart.js/dist/core/core.controller.d.ts b/node_modules/chart.js/dist/core/core.controller.d.ts
index dde4e6e..c4e16c5 100644
--- a/node_modules/chart.js/dist/core/core.controller.d.ts
+++ b/node_modules/chart.js/dist/core/core.controller.d.ts
@@ -26,6 +26,7 @@ declare class Chart {
     boxes: any[];
     currentDevicePixelRatio: any;
     chartArea: any;
+    crosshair: any;
     _active: any[];
     _lastEvent: import("../types/index.js").ChartEvent;
     _listeners: {};
diff --git a/node_modules/chart.js/dist/types/index.d.ts b/node_modules/chart.js/dist/types/index.d.ts
index 98bdf09..48e7752 100644
--- a/node_modules/chart.js/dist/types/index.d.ts
+++ b/node_modules/chart.js/dist/types/index.d.ts
@@ -481,6 +481,12 @@ export interface ActiveElement extends ActiveDataPoint {
   element: Element;
 }
 
+export interface Crosshair {
+  x: number | null,
+  y: number | null,
+  draw?: boolean
+}
+
 export declare class Chart<
   TType extends ChartType = ChartType,
   TData = DefaultDataPoint<TType>,
@@ -505,6 +511,7 @@ export declare class Chart<
 
   data: ChartData<TType, TData, TLabel>;
   options: ChartOptions<TType>;
+  crosshair: Crosshair;
 
   constructor(item: ChartItem, config: ChartConfiguration<TType, TData, TLabel> | ChartConfigurationCustomTypesPerDataset<TType, TData, TLabel>);
 
@@ -2937,8 +2944,14 @@ export interface TooltipItem<TType extends ChartType> {
   element: Element;
 }
 
+export interface CrosshairPluginOptions {
+  width?: number;
+  color?: string;
+}
+
 export interface PluginOptionsByType<TType extends ChartType> {
   colors: ColorsPluginOptions;
+  crosshair: CrosshairPluginOptions;
   decimation: DecimationOptions;
   filler: FillerOptions;
   legend: LegendOptions<TType>;
